index.html

```diff
@@ -131,48 +124,63 @@ <h3>Step 1: Type an address into the address bar</h3>
 
 				<section>
 					<h3>Step 2: Your computer performs a DNS lookup</h3>
```

Comment:
No longer mentioning DNS in the slide itself. How about "your computer looks up the website's IP address"?